docs/deploy.md

```diff
@@ -25,13 +25,6 @@ You can also deploy Voici as a standalone application so it can be hosted anywhe
 
 First install Voici by following the [installation instructions](install.md).
 
-You also need to install a kernel for the language you want to use. For example, to use Python, you can install the `jupyterlite-xeus-python` package.
```

Comment:
Should the docs mention the two `voici-core` and `voici` packages?

For reference the JupyterLite docs try to stick to mentioning `jupyterlite-core` because this is what provides the CLI: https://jupyterlite.readthedocs.io/en/stable/

Also the `jupyterlite` metapackage removed the dependency on `jupyterlite-pyodide-kernel` in https://github.com/jupyterlite/jupyterlite/pull/1147.

If Voici depends on `jupyterlite-xeus`, it would be nice to explain why.